core/src/main/scala/org/apache/spark/status/api/v1/api.scala

```diff
@@ -279,7 +292,9 @@ class StageData private[spark](
     val resourceProfileId: Int,
     @JsonSerialize(using = classOf[ExecutorMetricsJsonSerializer])
     @JsonDeserialize(using = classOf[ExecutorMetricsJsonDeserializer])
-    val peakExecutorMetrics: Option[ExecutorMetrics])
+    val peakExecutorMetrics: Option[ExecutorMetrics],
+    val taskMetricsDistributions: Option[TaskMetricDistributions],
+    val executorMetricsDistributions: Option[ExecutorMetricsDistributions])
```

Comment:
This is detected as a binary incompatibility.

Reply:
@AngersZhuuuu is this OK to 'break' as it's it's private[spark]? we just discovered that it was kind of accidentally excluded from a previous change between 3.0 -> 3.1. If so we can just retain the exclusion, just checking.